bin/fin

```diff
@@ -119,10 +119,29 @@ is_katacoda ()
 	[[ "$KATACODA" != "" ]]
 }
 
-is_github_actions () {
+is_github_actions ()
+{
 	[[ "$GITHUB_RUN_ID" != "" ]]
 }
 
+# Returns x86_64 / aarch64 (aka "old school")
+host_arch () {
+	case "$(uname -m)" in
+		x86_64) echo "x86_64" ;;
+		aarch64) echo "aarch64" ;;
+		* ) false;;
+	esac
+}
+
+# Returns amd64 / arm64 (aka "new style")
+host_arch_new () {
```

Comment:
The function is not used anywhere, can be deleted

Reply:
This was in use for Docker Compose v2-beta download URLs, then they switched to the other variant for the v2 release. There does not seem to be a single standard and we may need the "new style" version at some point with other tools. I'd rather have it there for completeness.